yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -57,6 +57,8 @@ type TargetDB interface {
 	QueryRow(query string) *sql.Row
 	Exec(query string) (int64, error)
 	WithTx(fn func(tx *sql.Tx) error) error
+	CheckTargetDBVersion() error
```

Comment:
This has no implementation in any of the structs? Remove it then?

Reply:
Yep removed